src/robot/rebot.py

```diff
@@ -200,6 +200,10 @@
                           match (starting from 1).
                           Examples: --tagstatlink mytag:http://my.domain:Title
                           --tagstatlink "bug-*:http://url/id=%1:Issue Tracker"
+    --autoexpandkeywords name:<pattern>|tag:<pattern> *
+                          Matching keywords will be initially expanded in log.html.
+                          For pattern matching rules see --removekeywords option.
+                          Example: --autoexpandkeywords name:CapturePageScreenshot
```

Comment:
`New in RF 3.1.2.` should be added somewhere.

Reply:
RF 3.2 note added

Reply:
updated with note new in RF 3.2